tests/scipy_stats_test.py

```diff
@@ -576,11 +576,11 @@ def testMultinomialLogPmf(self, shape, dtype):
     lax_fun = lsp_stats.multinomial.logpmf
 
     def args_maker():
-      x = np.rint(rng(shape, dtype[0])).astype(dtype[0])
+      x = np.rint(rng(shape, dtype[0]))
```

Comment:
Also it looks like `np.fix` or `np.around` will work too. Which do you think is best?